README.md

```diff
@@ -33,67 +32,72 @@ This off-chain application comprises 7 main goroutines: Monitor, Verifier, Vote
 ## Deployment
 
 ### Config
-1. Set your private key import method (via file or aws secret), deployment environment and gas limit.
-```
-  "greenfield_config": {
-    "key_type": "local_private_key" or "aws_private_key" depending on whether you are storing the keys on aws or locally in this json file
-    "aws_region": set this if you chose "aws_private_key"
-    "aws_secret_name": set this if you chose "aws_private_key"
-    "aws_bls_secret_name": set this if you chose "aws_private_key"
-    "private_key": set this if you chose "local_private_key"
-    "bls_private_key": set this if you chose "local_private_key" 
-    "rpc_addrs": [
-      "http://0.0.0.0:26750"
-    ],
-    "chain_id_string": chain id of the network, e.g., "greenfield_9000-121"
-    "gas_limit": transaction gas limit, e.g., 1000,
-    "fee_amount": transaction fees, e.g., "5000000000000",
-    "fee_denom": transaction fees denom, e.g., "BNB",
-    "no_simulate": simulate transaction, e.g., true
-    "deduplication_interval": skip events that were recently processed, e.g., 100
-  }
-```
 
-2. Set your log and backup preferences.
-```
-"log_config": {
-  "level": "DEBUG",
-  "filename": "log.txt",
-  "max_file_size_in_mb": 100 (file size threshold)  
-  "max_backups_of_log_files": 2 (backup count threshold)
-  "max_age_to_retain_log_files_in_days": 10 (backup age threshold)
-  "use_console_logger": true,
-  "use_file_logger": false,
-  "compress": false
-}
-```
+See [config.json](https://github.com/bnb-chain/bnb-chain-charts/blob/main/gnfd-challenger-testnet-values/values.yaml#L8). Reference for a complete testnet config file.
```

Comment:
Maybe, we should also have a section to let the users know 1) who should run it - only validators, 2) why they need to run it, and 3) the benefit of running it - reward.